Kartavya052000/Truvoye-frontend
# Kan 46 login and log out with auth token front end

###Summary
This PR introduces the functionality to set and remove authentication tokens as cookies upon user sign-in and sign-out respectively.

### How to Test
1. Sign up or log in to the application.
2. Verify that the token is stored as a cookie.
3. Sign out and ensure that the token cookie is removed.

### Note
This implementation uses the `react-cookie` library for managing cookies.
